vhdbuilder/packer/install-dependencies.sh

```diff
@@ -459,6 +461,52 @@ elif [ "$OS" = "$MARINER_OS_NAME" ] && [ "$OS_VERSION" = "3.0" ] && [ "$(isARM64
   installAndConfigureArtifactStreaming acr-mirror-azurelinux3 rpm
 fi
 
+should_install_device_plugin() {
+  # Require x86_64 (skip arm64)
+  if [ "$(isARM64)" -eq 1 ]; then return 1; fi
+
+  # Ubuntu 24.04 only
+  if [ "$OS" = "$UBUNTU_OS_NAME" ] && [ "${UBUNTU_RELEASE:-}" = "24.04" ]; then return 0; fi
+
+  # Azure Linux 3.0 only
+  if [ "$OS" = "$AZURELINUX_OS_NAME" ] && [ "${OS_VERSION}" = "3.0" ]; then return 0; fi
+
+  return 1
+}
+
+install_device_plugin() {
+  local pkg="${K8S_DEVICE_PLUGIN_PKG}"
+  echo "Installing ${pkg} on ${OS} (Ubuntu ${UBUNTU_RELEASE:-n/a} / Version ${OS_VERSION:-n/a})..."
```

Comment:
is `-n/a` a valid syntax ? 

also why are we printing Ubuntu and Version in the same log line ? aint those variable never set at the same time ?

Reply:
Syntax seems to be valid. 

And that's right: I've simplified the log line.